src/modules/rep.ts

```diff
@@ -182,30 +205,69 @@ export class RepModule extends Module {
 		aliases: ['leaderboard', 'lb'],
 		description: 'Reputation: See who has the most reputation',
 	})
-	async leaderboard(msg: Message) {
+	async leaderboard(msg: Message, @optional period: string = 'month') {
+		let periods = {
+			'rolling-hour': [
+				'Within the Last Hour',
```

Comment:
Less capital letters would be nice since
`Top 10 Reputation Within the Last Hour` reads a bit weird.